alot/widgets/colour_text.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import re
+from ..settings.const import settings
+
+
+def parse_text_colour(line):
+    """Return colour attribute as (attr, attr_focus)
+
+    :param str line: line of text to be parsed
+    :return: The theme attribute to apply
+    """
+    if settings.get('parse_quotes'):
+        return parse_quotes(line)
+
+
+def parse_quotes(line, max_quote_level=7):
+    """Search for quotes.
+    :param str line: The line of text to be parsed
+    :param int max_quote_level: Search for quotes up to that level
+
+    :return: Theming attribute, None if no quote are available
+    """
+    is_quoted, quote_colour = get_quote_colour(line, max_quote_level)
+
+    if is_quoted and quote_colour is not None:
+        return quote_colour
+    else:
+        return None
+
+
+def get_quote_colour(line, max_quote_level):
```

Comment:
It would be good to provide some tests for this code.

Reply:
Ok! I'll work on it.